docs/guide/web-simulation.md

```diff
@@ -324,6 +324,71 @@ This folder also contains a `server.sh` utility script to automatically start an
 
 Please make sure that the `WEBOTS_HOME` variable is set before running the simulation and session server scripts.
 
+#### Using Docker
+
+The simulation server can also be run in a Docker container.
+In this section we provide a sample Docker setup that could be used to run the simulation on localhost.
+
+You can use the following `Dockerfile` to build your Docker image.
+```
+FROM cyberbotics/webots:R2021a-ubuntu18.04
+
+RUN apt update
+RUN apt install -y nano psmisc firejail python3-pip
+RUN pip3 install websocket-client tornado nvidia-ml-py psutil requests distro
+RUN pip3 install -U pynvml --user
+ENV AUDIODEV=null
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DISPLAY=:99
+ENV LIBGL_ALWAYS_SOFTWARE=true
```

Comment:
Will that disable 3D acceleration?

Reply:
I will check.
I copied this from the already existing documentation.

Reply:
Yes, this disables hardware acceleration.
But I checked and it is no longer needed.